libs/game/systemmenu.ts

```diff
@@ -389,10 +396,17 @@ namespace scene.systemMenu {
 
     function initVolume() {
         const vol = settings.readNumber("#volume")
-        if (vol != null)
+        if (vol !== null)
```

Comment:
Are you sure read number doesn't return undefined?

Reply:
updated pr